lib/pagseguro/documents.rb

```diff
@@ -0,0 +1,27 @@
+module PagSeguro
+  class Documents
```

Comment:
I don't think a good idea now. I use only two methods out of the ruby box, like the *mass assignment* and *ensure_type*. Now I'm trying to keep the things more simple, without some dependecies. Maybe later... ty